extra/exports

```diff
@@ -65,6 +65,7 @@ box_index_get
 box_index_id_by_name
 box_index_iterator
 box_index_iterator_after
```

Comment:
Looks like it's not used anymore. Please drop it.

Reply:
Done.